source/gui/settingsDialogs.py

```diff
@@ -2415,6 +2415,27 @@ def __init__(self, parent):
 			["featureFlag", "cancelExpiredFocusSpeech"]
 		)
 
+		# Translators: This is the label for a group of advanced options in the
+		#  Advanced settings panel
+		label = _("Audio")
+		audioGroup = guiHelper.BoxSizerHelper(
+			parent=self,
+			sizer=wx.StaticBoxSizer(parent=self, label=label, orient=wx.VERTICAL)
+		)
+		sHelper.addItem(audioGroup)
+
+		# Translators: This is the label for a numeric control in the
+		#  Advanced settings panel.
+		label = _("Idle close delay (in ms)")
+		self.idleCloseDelaySpinControl = audioGroup.addLabeledControl(
+			label,
+			nvdaControls.SelectOnFocusSpinCtrl,
+			min=0,

```

Comment:
I believe there is logic to get this from config. Search for usage of `config.conf.getConfigValidation`